.github/workflows/deploy.yml

```diff
@@ -96,7 +96,7 @@ jobs:
           REDIS_PORT: 6379
           REDIS_DB: 0
           METADATA_BUCKET: hathor-explorer-metadata-mainnet
-          CORS_ALLOWED_ORIGIN: https://explorer.hathor.network
+          CORS_ALLOWED_REGEX: https?://([a-z0-9])*[.]*hathor[.]network
```

Comment:
This regex is different from the above. 
`https?://([a-z0-9])*[.]*hathor[.]network`
`https?://([a-z0-9]+[.])*hathor[.]network`

Reply:
I think I get it now. You made them different because in mainnet the address is only `explorer.hathor.netowrk`, right?

But we actually have this one too that I think we would want to support: `https://explorer.mainnet.hathor.network/`

Reply:
I guess it will be supported. I think even `https://hathor.network` will be supported

Reply:
Maybe `https?://([a-z0-9]*\.){0,5}hathor\.network` which allows the main domain and any subdomain up to 5 parts (I think we only use 3).

Reply:
I will use Andre suggestions as it contemplates

https://explorer.hathor.network
https://explorer.mainnet.hathor.network
https://hathor.network